### PR TITLE
Dataview Entry: Hide/Show toolbar along with location dataview

### PR DESCRIPTION
**Pull request by AlexanderGeere**
## Description
The toolbar elements should hide/show along with their associated dataview.

## GitHub Issue
[#1739](https://github.com/GEOLYTIX/xyz/issues/1739)

## Type of Change
- ✅ Bug fix (non-breaking change which fixes an issue)

## How have you tested this? 
Can be tested using the `Report Input` layer on `bugs_testing/ui_elements/workspace.json`. Click the checkbox that hides and shows the dataview. The toolbar should hide and show along with it. 

## Testing Checklist 
- ✅ Existing Tests still pass
- ✅ Ran locally on my machine

## Code Quality Checklist
- ✅ My code follows the guidelines of XYZ
- ✅ My code has been commented
- ✅ Documentation has been updated
- ✅ New and existing unit tests pass locally with my changes
- ✅ Main has been merged into this PR

### CODE REVIEW

File: lib/ui/Dataview.mjs [hunk 1]
```diff
@@ -331,10 +337,13 @@ function dataviewToolbar(_this) {
     .map((key) => mapp.ui.utils[_this.dataview]?.toolbar[key]?.(_this))
     .filter((item) => !!item);
 
+  // Create an element for the toolbar buttons
+  _this.btn_row = mapp.utils.html.node`<div class="btn-row">${toolbarElements}</div>`
```
Comment (RobAndrewHurst) [inline lib/ui/Dataview.mjs:341]: Can we use camelCame 🐫 as I think it's the style we want to start perpetuating

**Review by RobAndrewHurst — DISMISSED** [review]

File: lib/ui/Dataview.mjs (hunk 1 above)
Comment (AlexanderGeere) [reply lib/ui/Dataview.mjs]: Resolved in [0f21251](https://github.com/GEOLYTIX/xyz/pull/1740/commits/0f2125196d797256a7966b36d67ffe9d518cc85b)

**Comment by simon-leech** [comment]
Done one commit to fix the camelCase and ensure that btnRow is initially display none. 
Just adding tests for this now

**Comment by simon-leech** [comment]
Just pushed some tests to test the show hide methods and that the btnRow is correctly hidden when these methods are called too

**Review by simon-leech — APPROVED** [review]

**Pull request merged**